### PR TITLE
Style fixes for Playground

### PR DESCRIPTION
**Pull request by ikesyo**

### CODE REVIEW

File: ReactiveCocoaTests/Swift/SignalProducerSpec.swift [hunk 1]
```diff
@@ -1409,9 +1409,9 @@ class SignalProducerSpec: QuickSpec {
 			}
 
 			describe("disposal") {
-				var completeOuter: (Void -> Void)!
-				var disposeOuter: (Void -> Void)!
-				var execute: (FlattenStrategy -> Void)!
+				var completeOuter: (() -> ())!
+				var disposeOuter: (() -> ())!
+				var execute: (FlattenStrategy -> ())!
```
Comment (RuiAAPeres) [inline ReactiveCocoaTests/Swift/SignalProducerSpec.swift]: I have to be honest, I hate `()` instead of `Void`, it makes it much more difficult to read. 👎

Comment (ikesyo) [inline ReactiveCocoaTests/Swift/SignalProducerSpec.swift]: I prefer `Void` too, but that was previously rejected in #
2059.

Comment (RuiAAPeres) [inline ReactiveCocoaTests/Swift/SignalProducerSpec.swift]: That issue was open almost one year ago, maybe we could re-open this discussion?

Comment (mdiep) [inline ReactiveCocoaTests/Swift/SignalProducerSpec.swift]: Let's just be consistent in this PR. I don't have a strong opinion and would be open to changing to `Void`. (Particularly if we're going to match the Swift stdlib style.) But let's do that consistently across the codebase if we're going to do it.

Comment (RuiAAPeres) [inline ReactiveCocoaTests/Swift/SignalProducerSpec.swift]: 👍

**Comment by mdiep** [comment]
✨

**Comment by ikesyo** [comment]
Thanks @RuiAAPeres and @mdiep. I'll open a separate PR for `Void` return types.

**Pull request merged**